de.fu_berlin.inf.dpp.ui.frontend/html/src/components/App.jsx

```diff
@@ -1,19 +1,19 @@
-import React from 'react'
-import $ from 'jquery'
 import { inject, observer } from 'mobx-react'
 import { views } from 'Constants'
-import MainView from './MainView'
+import $ from 'jquery'
 import AddContactView from './AddContactView'
-import StartSessionWizardView from './StartSessionWizardView'
-import ConfigurationWizardView from './ConfigurationWizardView'
 import BasicWidgetTestView from './BasicWidgetTestView'
+import ConfigurationWizardView from './ConfigurationWizardView'
+import MainView from './MainView'
+import React from 'react'
+import StartSessionWizardView from './StartSessionWizardView'
 
 const viewComponents = {
   [views.MAIN]: MainView,
   [views.ADD_CONTACT]: AddContactView,
   [views.START_SESSION_WIZARD]: StartSessionWizardView,
   [views.CONFIGURATION_WIZARD]: ConfigurationWizardView,
-  [views.BASIC_WIDGET_TEST]: BasicWidgetTestView
+  [views.BASIC_WIDGET_TEST]: BasicWidgetTestView,
```

Comment:
I went through the linter settings and this is how the linter would want it.

but it does not make sense to allow trailing commas, in older version of web browser (e.g: IE8) trailing commas will cause an exception, I suggest that the settings of the linter gets changed and the rule to comma-dangle should be set to "never"

Reply:
Should I change the linting rules in this PR or is this a topic of another PR? Is there a need to involve a wider range of people in this decision?

Reply:
Nevermind. I read your request.